src/SmartCommander/Assets/Resources.Designer.cs

```diff
@@ -626,13 +635,5 @@ public static string Zip {
                 return ResourceManager.GetString("Zip", resourceCulture);
             }
         }
-        /// <summary>
-        ///   AllowOnlyOneInstance
-        /// </summary>
-        public static string AllowOnlyOneInstance {
-            get {
-                return ResourceManager.GetString("AllowOnlyOneInstance", resourceCulture);
-            }
-        }
     }
```

Comment:
It seems the changes in this file can be reverted?

Reply:
its auto-sorting of generated files, aplies automatically in VS